### PR TITLE
Go.mod

### PR DESCRIPTION
**Pull request by EItanya**
BOT NOTES: 
resolves https://github.com/solo-io/solo-kit/issues/326

### CODE REVIEW

File: pkg/api/v1/clients/file/resource_client_test.go [hunk 1]
```diff
@@ -22,13 +22,13 @@ var _ = Describe("Base", func() {
 		client *ResourceClient
 		tmpDir string
 	)
-	BeforeEach(func() {
+	JustBeforeEach(func() {
```
Comment (yuval-k) [inline pkg/api/v1/clients/file/resource_client_test.go]: why is this needed?

File: pkg/code-generator/codegen/generator.go [hunk 2]
```diff
@@ -141,7 +141,7 @@ func generateFilesForResourceGroup(rg *model.ResourceGroup) (code_generator.File
 	} {
 		content, err := generateResourceGroupFile(rg, tmpl)
 		if err != nil {
-			return nil, errors.Wrapf(err, "internal error: processing %template '%v' for resource group %v failed", tmpl.ParseName, rg.Name)
+			return nil, errors.Wrapf(err, "internal error: processing %template '%s' for resource group %v failed", tmpl.ParseName, rg.Name)
```
Comment (yuval-k) [inline pkg/code-generator/codegen/generator.go]: what's the "%t"?

Comment (EItanya) [reply pkg/code-generator/codegen/generator.go]: Hahaha
That’s been there’s while I guess
Probably a mistake

File: pkg/api/v1/clients/file/resource_client_test.go (hunk 1 above)
Comment (EItanya) [reply pkg/api/v1/clients/file/resource_client_test.go]: It was failing without it
